src/arithmetic/agg_funcs.c

```diff
@@ -201,7 +201,9 @@ typedef struct {
 
 int __agg_countStep(AggCtx *ctx, SIValue *argv, int argc) {
     __agg_countCtx *ac = Agg_FuncCtx(ctx);
-    ac->count += argc;
+    for (int i = 0; i < argc; i ++) {
+        if (!SIValue_IsNullPtr(&argv[i])) ac->count ++;
```

Comment:
Count can only handle a single argument,
and so this loop is invalid, we should verify that argc == 1
and remove the loop,

if (!SIValue_IsNullPtr(&argv[0])) ac->count ++;

Neo4j: 
"Too many parameters for function 'count' (line 1, column 18 (offset: 17)) "match (n) return count(n,n)" ^